evaluation/degradation_eval.py

```diff
@@ -1,62 +1,77 @@
 import pathlib
 import random
 import csv
+import simpleaudio as sa
 from datetime import datetime
-from playsound import playsound
 
+
+ONE_MINUTE_FILES_DIRECTORY_NAME = 'audio/1min_files'
 AUDIO_FILES_DIRECTORY_NAME = 'audio/evaluation_samples'
 EVAL_REPORTS_DIRECTORY_NAME = 'eval_reports'
 EVAL_REPORT_FILE_STRING = 'eval_report_'
 
 
 def process_examples():
     print(audio_file_path)
-    for audio_example_pair in audio_file_path.glob('*'):
-        print(f'\nNow evaluating example at {audio_example_pair}')
-        process_example_pair(audio_example_pair)
+    for test_set in audio_file_path.glob('*'):
+        for audio_example in test_set.glob('*'):
+            process_example_pair(audio_example)
 
 
-def process_example_pair(audio_example_pair):
+def process_example_pair(audio_example):
     audio_sample_list = []
-    for audio_example in audio_example_pair.glob('*'):
-        print(audio_example)
-        modified = "modified" in str(audio_example)
-        audio_sample_list.append((audio_example, modified))
+    # add the modified file to the sample list
+    audio_sample_list.append((audio_example, True))
```

Comment:
```suggestion
    audio_sample_list = [(audio_example, True)]
```
PyCharm empfiehlt: Zeile 22 weg und stattdessen der Vorschlag

Reply:
Nicht länger aktuell, da randomisiertes Test-Setup erstellt wird.